src/sinks/humio/metrics.rs

```diff
@@ -40,11 +43,16 @@ pub struct HumioMetricsConfig {
     transform: MetricToLogConfig,
 
     /// The Humio ingestion token.
+    #[configurable(metadata(
+        docs::examples = "${HUMIO_TOKEN}",
+        docs::examples = "A94A8FE5CCB19BA61C4C08"
+    ))]
     token: SensitiveString,
 
     /// The base URL of the Humio instance.
     #[serde(alias = "host")]
-    pub(super) endpoint: Option<String>,
+    #[serde(default = "default_endpoint")]
+    pub(super) endpoint: String,
```

Comment:
same question about example, what is required.

also, what happens if additions to the URI are provided? eg

http://somehost:1234/foo?bar

Reply:
That is all covered in the word `base` in base URL, no?

Reply:
My comment is partially motivated by config structs where we have address/endpoint, and call out things like "the address must include a port" , or "Both IP address and hostname are accepted formats". Like Spencer is saying, the more clear the requirments are in the documentation, the less guesswork/trial error the user has to do.

Reply:
👍 the shared type crossed my mind too